apps/src/netsim/NetSimLogBrowserTable.jsx

```diff
@@ -232,7 +232,7 @@ function wrappedSortable(defaultSortable) {
     const {className: originalClassName} = originalProps;
     return {
       ...newProps,
-      originalClassName,
+      className: originalClassName,
```

Comment:
This fixes a warning for unknown properties on DOM elements, [introduced in React 15.2.0](https://github.com/facebook/react/blob/master/CHANGELOG.md#react-dom-2).